src/coreclr/jit/gentree.h

```diff
@@ -1635,7 +1634,7 @@ struct GenTree
 
     bool OperIsSsaDef() const
     {
-        return OperIs(GT_ASG, GT_CALL) || OperIsLocalStore();
+        return OperIs(GT_CALL) || OperIsLocalStore();
```

Comment:
Probably one of the cases where reordering would be beneficial, though presumably native PGO gets us that anyway.

Reply:
Will make a note on that. It could also just be deleted from a number of places as they use `DefinesLocal`, which does the same checks anyway.